### ISSUE TITLE
[med] Keyboard Navigation: Many Pages: Timing not adjustable for toast messages

### ISSUE THREAD

**Issue by ogumen** [Expensify/App#5679]
## Action Performed:
Prerequisites:
- User has created personal account and is logged into same.
- User has Concierge chat in chat history.

1. Using Chrome, open https://staging.new.expensify.com/new/group
2. Tab to the Concierge chat in the history and press enter.
3. Tab to focus the phone icon at the top of the chat window and press Enter key.
4. Leave any of the 3 fields blank, then tab to focus the "Call me" button and press Enter key.
5. Note the resulting error toast message that appears briefly before disappearing.

## Expected Result:
The user must have a way to extend or disable the timing on the message so they have time to read it.

## Actual Result:
The message is displayed only for approx 2-2.5 seconds. Not long enough for many users to read before it disappears.

HTML:
`<div tabindex="0" class="css-1dbjc4n"><div class="css-1dbjc4n r-1awozwy r-18u37iz" style="background-color: rgb(11, 27, 52); border-radius: 8px; box-shadow: rgb(0, 0, 0) 0px 0px 0px; justify-content: space-between; padding: 20px; -webkit-box-pack: justify;"><div dir="auto" class="css-901oao" style="color: rgb(255, 255, 255); font-family: GTAmericaExp-Regular; font-size: 15px; line-height: 20px; width: 90%;">Please provide both a first and last name so our Guides know how to address you!</div><svg xmlns="http://www.w3.org/2000/svg"
viewBox="0 0 20 20" width="20" height="20" fill="#03d47c"><path d="M17.1 5.5l-.9-.9c-.2-.2-.6-.2-.9 0l-7.6 7.6-3.1-3c-.2-.2-.6-.2-.8 0l-.9.9c-.2.2-.2.6 0 .9L6 14.1l1.4 1.4c.2.2.6.2.9 0l1.4-1.4 7.6-7.6c.1-.4.1-.7-.2-1z"></path></svg></div></div>`

Other occurrences:
- The same issue is seen for the confirmation message displayed after saving any profile details in Settings > Profile overlay.
- Also seen for the confirmation message displayed after adding or updating the PayPal.me account.

## Workaround:
No workaround

## Area issue was found in:
Request a call overlay, Profile overlay, Add payment method flow, 

## Failed WCAG checkpoints
2.2.1

## User impact:
Many users with sight, cognitive and other impairments will not have enough time to read the error or success message.

## Suggested resolution:
For field and form input errors, it is best not to use pop-ups or toasts to convey the error. Make the error text static on the form itself. Make it persistent visually until the error has been corrected.
Use dialogs with a close button to implement the confirmation messages, so that users can close the dialog once they have read the displayed message.
https://www.w3.org/WAI/WCAG21/Understanding/timing-adjustable.html

## Platform:
Where is this issue occurring?

- Web

**Version Number:**
**Reproducible in staging?:**Yes
**Reproducible in production?:**
**Notes/Photos/Videos:** Any additional supporting documentation
**Issue reported by:**
![Bug5187663_Keyboard_Navigation-Chat-Timing_not_adjustable_for_error_toast_messages](https://user-images.githubusercontent.com/88733897/136207487-02073f86-a835-4cee-aa53-cf3e2cdc49a7.png)
![Bug5187663_Payments_-_Timing_not_adjustable_for_toast_message](https://user-images.githubusercontent.com/88733897/136207508-086b6b9f-7c75-4d6f-88d2-014c53307a46.png)
![Bug5187663_Profile_-_Timing_not_adjustable_for_toast_message](https://user-images.githubusercontent.com/88733897/136207534-2cf82b2c-06cd-4b34-8d60-99660af2703b.png)

https://user-images.githubusercontent.com/88733897/136207569-978ae21c-9668-4aea-b3e8-aa6d33da3625.mp4

[View all open jobs on GitHub](https://github.com/Expensify/App/issues?q=is%3Aopen+is%3Aissue+label%3A%22Help+Wanted%22)
